charts/opentelemetry-operator/templates/admission-webhooks/operator-webhook.yaml

```diff
@@ -1,5 +1,5 @@
 {{- if and (.Values.admissionWebhooks.create) (not .Values.admissionWebhooks.certManager.enabled) }}
-{{- $altNames := list ( printf "%s.%s" "opentelemetry-operator-webhook-service" .Release.Namespace ) ( printf "%s.%s.svc" "opentelemetry-operator-webhook-service" .Release.Namespace ) -}}
+{{- $altNames := list ( printf "%s-webhook-service.%s" (include "opentelemetry-operator.name" .) .Release.Namespace ) ( printf "%s-webhook-service.%s.svc" (include "opentelemetry-operator.name" .) .Release.Namespace ) -}}
```

Comment:
@Allex1 should we use the `fullname` template here instead

Reply:
sure, that seems more appropriate

Reply:
@TylerHelmuth 
I think that the alt name should match whatever is set as the webhook service name, eg. see
https://github.com/open-telemetry/opentelemetry-helm-charts/blob/main/charts/opentelemetry-operator/templates/admission-webhooks/operator-webhook.yaml#L34
```
name: {{ template "opentelemetry-operator.name" . }}-webhook-service
```
Here, opentelemetry-operator.name is used, not fullname